test/jest/SwapClientManager.spec.ts

```diff
@@ -99,7 +109,7 @@ describe('Swaps.SwapClientManager', () => {
 
   test('it initializes lnd-ltc, lnd-btc and raiden', async () => {
```

Comment:
think we can also consider asserting
```
    expect(swapClientManager.raidenClient.tokenAddresses.size).toEqual(1);
    expect(swapClientManager.raidenClient.tokenAddresses.get('WETH')).not.toBeUndefined();
```
to test that the tokenAddress does get set.

Reply:
And remove line 43 that currently adds this value via mock: `tokenAddresses.set('WETH', '0x1234');`

Reply:
Done, thanks!